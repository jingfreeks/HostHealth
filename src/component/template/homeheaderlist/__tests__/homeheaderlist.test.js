import React from 'react';
import {render} from '@testing-library/react-native';
import HomeHeaderlist from '../homeheaderlist';

describe('Home Header list Template Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<HomeHeaderlist />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
