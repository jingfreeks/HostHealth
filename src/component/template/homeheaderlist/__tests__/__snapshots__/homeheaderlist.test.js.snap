// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Header list Template Component Should work as expected to get snapshot 1`] = `
[
  <View
    style={
      [
        {
          "alignself": "stretch",
          "borderColor": "#99E1E1",
          "borderRadius": 10,
          "borderWidth": 2,
          "flexDirection": "row",
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "marginBottom": 10,
            "marginLeft": 10,
            "marginTop": 10,
          },
        ]
      }
    >
      <Image
        resizeMode="stretch"
        size={50}
        source={
          {
            "testUri": "../../../src/assets/htimecard.png",
          }
        }
        style={
          [
            {
              "height": 117.70588235294117,
              "width": 98.08823529411764,
            },
          ]
        }
      />
    </View>
    <View
      style={
        [
          {
            "alignSelf": "center",
            "flexBasis": 0,
            "flexGrow": 3,
            "flexShrink": 1,
            "justifyContent": "center",
            "marginBottom": 10,
            "marginTop": 10,
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#0e3f6c",
              "fontFamily": "Avenir",
              "fontSize": 20,
              "fontWeight": "900",
            },
          ]
        }
      >
        Timecards
      </Text>
      <Text
        style={
          [
            {
              "color": "#0e3f6c",
              "fontFamily": "Avenir",
              "fontSize": 12,
              "fontWeight": "400",
            },
          ]
        }
      >
        Easily upload images for 
your timecards for quick processing.
      </Text>
    </View>
  </View>,
  <View
    style={
      [
        {
          "alignself": "stretch",
          "borderColor": "#99E1E1",
          "borderRadius": 10,
          "borderWidth": 2,
          "flexDirection": "row",
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "marginBottom": 10,
            "marginLeft": 10,
            "marginTop": 10,
          },
        ]
      }
    >
      <Image
        resizeMode="stretch"
        size={50}
        source={
          {
            "testUri": "../../../src/assets/certificate.png",
          }
        }
        style={
          [
            {
              "height": 117.70588235294117,
              "width": 98.08823529411764,
            },
          ]
        }
      />
    </View>
    <View
      style={
        [
          {
            "alignSelf": "center",
            "flexBasis": 0,
            "flexGrow": 3,
            "flexShrink": 1,
            "justifyContent": "center",
            "marginBottom": 10,
            "marginTop": 10,
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#0e3f6c",
              "fontFamily": "Avenir",
              "fontSize": 20,
              "fontWeight": "900",
            },
          ]
        }
      >
        Credentialing
      </Text>
      <Text
        style={
          [
            {
              "color": "#0e3f6c",
              "fontFamily": "Avenir",
              "fontSize": 12,
              "fontWeight": "400",
            },
          ]
        }
      >
        Add information for backgournd
check and 19. (Links to Essium)
      </Text>
    </View>
  </View>,
]
`;
